### PR TITLE
Fix/improve sensitivity log

### LINKED ISSUE #474: Recall optimal capacties when starting sensitivity analysis
When launching sensitivity analysis, it could be interesting to print the results of the Benders run (optimal investment capacities) so that the user can analyse at a glance the output of the sensitivity analysis with regard to the optimal capacities

### PR DESCRIPTION
**Pull request by tbittar**
Closes #474

### CODE REVIEW

**Review by a-zakir — APPROVED** [review]

File: src/cpp/sensitivity/SensitivityLogger.cpp [hunk 1]
```diff
@@ -14,14 +14,47 @@ void SensitivityLogger::display_message(const std::string& msg) {
...
+  _stream << indent_1 << "Best capex = "
+          << xpansion::logger::commons::create_str_million_euros(benders_capex)
+          << MILLON_EUROS << std::endl
+          << std::endl;
+}
+
+void SensitivityLogger::log_benders_solution(
+    const std::map<std::string, double>& benders_solution) {
+  _stream << indent_1 << "Best investment solution = " << std::endl;
+  for (const auto& kvp : benders_solution) {
```
Comment (JasonMarechal25) [inline src/cpp/sensitivity/SensitivityLogger.cpp]: You can use structured binding. You can even use better names than "key" and "value", like "candidate" and "investement" or what is proper in this case.
```suggestion
  for (const auto& [key, value] : benders_solution) {
      _stream << indent_1 << indent_1 << key << ": "
            << xpansion::logger::commons::create_str_mw(value) << MW
```

File: src/cpp/sensitivity/include/SensitivityILogger.h [hunk 2]
```diff
@@ -3,17 +3,19 @@
 
 #include <string>
 
+#include "SensitivityInputReader.h"
```
Comment (JasonMarechal25) [inline src/cpp/sensitivity/include/SensitivityILogger.h:6]: Why is this include necessary?

File: src/cpp/sensitivity/include/SensitivityWriter.h [hunk 3]
```diff
@@ -4,6 +4,7 @@
 
 #include <filesystem>
 
+#include "SensitivityInputReader.h"
```
Comment (JasonMarechal25) [inline src/cpp/sensitivity/include/SensitivityWriter.h:7]: Same as previously, why is the include necessary ?

File: src/cpp/sensitivity/include/SensitivityILogger.h (hunk 2 above)
Comment (tbittar) [reply src/cpp/sensitivity/include/SensitivityILogger.h:6]: Useful to know the SensitivityinputData structure, should we put it in a separate file ?

File: src/cpp/sensitivity/include/SensitivityWriter.h (hunk 3 above)
Comment (tbittar) [reply src/cpp/sensitivity/include/SensitivityWriter.h:7]: See answer above

File: src/cpp/sensitivity/SensitivityLogger.cpp (hunk 1 above)
Comment (tbittar) [reply src/cpp/sensitivity/SensitivityLogger.cpp]: Done

**Pull request merged**